src/client/search/AddSearchTerm.js

```diff
@@ -22,12 +22,15 @@ let AddSearchTerm = ({ dispatch }) => {
         onBlur={() => {
           $('#searchTermContainer').slideUp('fast');
         }}
+
         onKeyDown={(e) => {
           if (e.keyCode === 13) {
             dispatch(addSearchTerm(nextSearchTermId++, e.target.value));
+            dispatch(invalidateFeedResults());
```

Comment:
This is fine for now, however I envisage that this dispatch will happen in SearchQueryTerms. Haven't thought about it completely yet but the way I see it there are two options:
1. We dispatch an action every place that we add searchQueryTerms
2. We dispatch 1 action for when the SearchQueryTerms component recieves a new term. So it's decoupled from the search filters and AddSearchTerm components.

Thoughts @Winwardo ?